docs/ABOUT.md

```diff
@@ -1,4 +1,6 @@
-[OCaml](https://ocaml.org/) is an industrial strength programming language supporting functional, imperative and object-oriented styles. 
-It is used as a "teaching language" by a lot of institutions. 
-OCaml has a rich and powerful [type system](https://en.wikipedia.org/wiki/Type_system) that allows the compiler to catch some mistakes early on saving the developer a huge amount of frustration. 
-[Type inference](https://en.wikipedia.org/wiki/Type_inference) relieves the developer from the tedium of providing all the type signatures by themselves.
+[OCaml](https://ocaml.org/) is an industrial strength programming language supporting functional, imperative and object-oriented styles - but don't let this put you off, as it's used as a "teaching language" by a lot of institutions. 
```

Comment:
I don't think there's anything in the paragraph that should put anyone off. Perhaps say something along the lines of "don't worry if you're not familiar with all these styles". 
Teaching language does not need to be in quotes. 
Maybe give a few examples of the institutions.